requirements/docs.txt

```diff
@@ -1,9 +1,9 @@
-sphinx>=4.0,<4.1
-myst-parser
+sphinx>=4.0
```

Comment:
we could restrict to < 5.0 to avoid surprises when they release it and we don't know about it. We had problems transitioning from 3.0 to 4.0, especially because some of the plugins were not ready by the time 4.0 came out.

Reply:
Yeah, I like that idea. Done :smiley: